.github/issue_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- GitHub is for bugs, feature and enhancement requests, and discussion on developing Service Fabric, as opposed to using it.
+If you need help or have any questions regarding using Service Fabric, you can ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-service-fabric)
+-->
+> If you know the area of code this belongs to, please select an appropriate tag.
+> Otherwise, select `triage-needed`
+
+**Bug or Enhancement:**
+ 
+ ---- 
+ 
+> Questionnaire for bug reports.  Feel free to delete if filing an enhancement
+
+**Describe the issue:**
+
+**Describe as best as possible how to reproduce the issue:**
+
+> If found from a deployment, please include the following
+
+**Service Fabric Version:**
+
+**Guest OS version:**
```

Comment:
Wonder if we should have a script that people can run to grab the appropriate data.